### DISCUSSION TITLE
Icon request  - Bold  rating star

### DISCUSSION THREAD

**Discussion by alexc957** [twilio-labs/paste#3269] [Request: Icon] [answered]
### Description

Hello Team, I see that currently paste has this rating star 
<img width="699" alt="image" src="https://github.com/twilio-labs/paste/assets/23362665/e9c62850-523a-4e89-8c43-a5885950ff47">

However it would be good to have a bold rating star and the hability to change its fill color. 

I hope someone can help me! 

### Link to icon

https://www.streamlinehq.com/icons/streamline-bold/social-medias-rewards-rating/rating/rating-star

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by nkrantz** [comment]
Hi @alexc957 - could you please share a screenshot or prototype of how this will be used? Chatting with the team about the addition. Thanks!

**Reply by alexc957** [reply]
Hi @nkrantz, I am developing a rating widget, I want to use that icon when the user selects a rating 
currently I am using the jsx that I got from streamline, but maybe adding that icon would be awesome.

**Reply by nkrantz** [reply, marked as answer]
Hi again! We'll need some more data points before we add the new icons to our system, but I'm sharing the SVG of the filled in star icon for you to use in your ratings widget. Will you also need the SVG of the bold star icon?


![StarFilled](https://github.com/twilio-labs/paste/assets/75342690/44a62f15-df43-4916-80ef-70a16685a507)

We'll also be sure to let you know if we do have more requests for these icons and end up adding them to our system in the future!

**Reply by alexc957** [reply]
Hi @nkrantz , no worries I am using the svg right now, if adding the icon the library takes too much time, then I will just use the svg as a react component.
